frontend/src/redux/user/actions.ts

```diff
@@ -58,6 +63,26 @@ export const register = createAsyncThunk<
   }
 });
 
+export const modifyUser = createAsyncThunk<
+  boolean,
+  UserModifyRequest,
+  { rejectValue: AxiosError }
+>('/user/patchUser', async (userPatch: UserModifyRequest, thunkAPI) => {
+  try {
+    if (userPatch.deleteUser) {
+      if (await api.deleteUser(userPatch)) {
+        return true;
+      }
+    } else if (await api.patchUser(userPatch)) {
```

Comment:
```suggestion
      return await api.deleteUser(userPatch));
    }
    if (await api.patchUser(userPatch)) {
```

Would be clearer to just return the result directly, since the `false` path doesn't do anything more